rotkehlchen/assets/asset.py

```diff
@@ -81,12 +81,17 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0x6F919D67967a97EA36195A2346d9244E60FE0dDB'): 'BLOC',
     strethaddress_to_identifier('0xc528c28FEC0A90C083328BC45f587eE215760A0F'): 'EDR',
     strethaddress_to_identifier('0xfAE4Ee59CDd86e3Be9e8b90b53AA866327D7c090'): 'CPC',
+    # Tokenized coinbase in bittrex
+    'COIN-2': 'COIN',
 }
 
 WORLD_TO_FTX = {
     strethaddress_to_identifier('0x50D1c9771902476076eCFc8B2A83Ad6b9355a4c9'): 'FTT',
     strethaddress_to_identifier('0xcca0c9c383076649604eE31b20248BC04FdF61cA'): 'ASD',
     'SOL-2': 'SOL',
+    'COIN': 'COIN',
```

Comment:
Strange that we need to have 2 entries for the same thing, once in bittrex and then again in FTX

Reply:
Coingecko has two listings from this. I believe this assets can't leave the exchange and then it makes sense for them to be two different assets since the price will be different (?)

Reply:
yeah saw that too. It's really weird and definitely won't scale.